### PR TITLE
Andrew Hill HW1

### PR DESCRIPTION
**Pull request by andrewchill25**

### CODE REVIEW

**Review by hibyeun — COMMENTED** [review]
Andrew did a good job inserting headings, bold, list, and a link. The arrangement of the content is clear and properly done.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+---
+I grew up in southern Mississippi and went to college at **Louisiana State University**. After college, I ran an outreach program through Baton Rouge Parks and Recreation aimed at increasing engagement with underserved members of the community. 
+
+My current interests include:
+* Urban land use policies
+* Architecture
+* Historic preservation 
+* Community engagement 
+
+My current work is on the [Pullman Neighborhood](https://www.nps.gov/pull/index.htm) on the South Side of Chicago
```
Comment (nnickels) [inline README.md:13]: Nice work including a link.

Comment (nnickels) [reply README.md:13]: Evaluation HW-01 for Andrew Hill
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X    |              |            |
| **Coding strategy**         |     X    |              |            |
| **Presentation**            |           |       X     |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great work on the first homework - clean, simple code that produces a clear README.md with a number of different markdown elements.
* You had a few comments on your pull request, but should include a slightly more in depth description of your changes and workflow, either in the README or as a longer comment on your pull request.

**Comment by marieakerman** [comment]
Evaluation HW-01 for Andrew Hill
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Code is clear and succinct, demonstrating aptitude with several skills.
* Nice use of bullet points and link to the Pullman website!
* The format of the headers was a bit confusing. It seemed like they were clickable links, but when I clicked them they did not lead anywhere. This may just have been a header format I am unfamiliar with.

**Review by marieakerman — COMMENTED** [review]
Submitting this again because I think I did it wrong the first time!

Evaluation HW-01 for Andrew Hill
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Code is clear and succinct, demonstrating aptitude with several skills.
* Nice use of bullet points and link to the Pullman website!
* The format of the headers was a bit confusing. It seemed like they were clickable links, but when I clicked them they did not lead anywhere. This may just have been a header format I am unfamiliar with.

**Pull request closed without merging**